docs/reference/transform/painless-examples.asciidoc

```diff
@@ -327,3 +327,63 @@ the buckets you want to use for the variable. In this particular case, `min` and
 `max` are variables mapped to `time_frame.gte.value` and `time_frame.lte.value`.
 <3> Finally, the script substracts the start date of the session from the end 
 date which results in the duration of the session.
+
+
+[discrete]
+[[painless-count-http]]
+==== Counting HTTP responses by using scripted metric aggregation
+
+You can count the different HTTP response types in a web log data set by using 
+scripted metric aggregation as part of the {transform}. The example below 
+assumes that the HTTP response codes are stored in the `response` field of the 
+documents.
+
+[source,js]
+--------------------------------------------------
+"aggregations": { <1>
```

Comment:
we omit "aggregations" on the other example. I think we should be consistent. However I like it, so I suggest to add "aggregations" on the other example.